CHANGES.md

```diff
@@ -300,6 +300,9 @@
 
 ### Bug Fixes and Minor Changes
 
+  * `XMonad.Prompt.XMonad`
+    - Adds `xmonadPromptCT` which allows you to create an XMonad prompt with a custom title.
+
   * Add support for GHC 9.0.1.
```

Comment:
I feel like this should come before any explicit modules, lest it gets lost

Reply:
Do you mean the GHC 9.0.1 support should go first in the log?

Reply:
Yes, exactly

Reply:
ok fixed!